Update_autoprotocol

danbryce opened the pull request:
Adds support for autoprotocol specialization of multi-color-calibration protocol.

bbartley reviewed (approved):
Is it possible to re-enable test_opentrons::TestProtocolEndToEnd::test_create_protocol?  It looks like the corresponding rubric file was updated, so does that mean the test should pass now?

I have some updates to enable the mix operations, but will submit those in a separate PR.

danbryce commented:
The opentrons test was failing for me because of a change to the container lookup that caused the BehaviorSpecialization code to differ from ` main`.  I changed it back to match `main`.  

I don't know that we need all the mix operations.  I looked at the autoprotocol and noticed that each transfer involves a single pipetting from the source, but then several at the destination.  These should mix the destination.  I changed the serial dilution to include an additional transfer from the source to source at the start so that the first column is mixed.  The subsequent transfers will mix each destiation well.
